src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -165,53 +165,6 @@ private static ServerExclusions CreateSettings()
             };
         }
 
-        [TestMethod]
-        public async Task DownloadQualityProfile_Success()
```

Comment:
This test is a subset of the modified test below i.e. it doesn't add anything in terms of scenario coverage, so I deleted it.